tests/adapter/dbt/tests/adapter/constraints/test_constraints.py

```diff
@@ -23,8 +23,10 @@
     my_model_incremental_wrong_name_sql,
```

Comment:
if you haven't already - let's open PRs in adapter repos to ensure existing tests pass + we inherit the new ones.